src/test/groovy/graphql/ScalarsStringTest.groovy

```diff
@@ -86,24 +85,15 @@ class ScalarsStringTest extends Specification {
     }
 
     @Unroll
-    def "String parseValue throws exception for non-String values"() {
-        when:
-        Scalars.GraphQLString.getCoercing().parseValue(literal, GraphQLContext.default, Locale.default)
-        then:
-        def ex = thrown(CoercingParseValueException)
+    def "String parseValue can parse non-String values"() {
+        expect:
+        Scalars.GraphQLString.getCoercing().parseValue(value, GraphQLContext.default, Locale.default) == result
 
         where:
-        literal      | _
-        123          | _
-        true         | _
-        customObject | _
+        value        | result
+        123          | "123"
+        true         | "true"
+        customObject | "foo"
```

Comment:
As all Java objects must have a `toString` method, there will always be a result for `parseValue` with the looser coercion.